plugins/chat/assets/javascripts/discourse/services/chat-subscriptions-manager.js

```diff
@@ -123,6 +140,34 @@ export default class ChatSubscriptionsManager extends Service {
     });
   }
 
+  @bind
+  _onKickFromChannel(busData) {
```

Comment:
Yes I realise this is not ideal how this is interacting with the UI -- I can refactor this once we extract the message actions and "panels" out as per https://github.com/discourse/discourse/pull/20324